### ISSUE TITLE
androidenv.composeAndroidPackages with toolsVersion 26.1.1 can't find libfontconfig.so.1 and fails

### ISSUE THREAD

**Issue by terlar** [NixOS/nixpkgs#54474]
## Issue description

When using specifying the `toolsVersion = "26.1.1"` with `androidenv.composeAndroidPackages` it fails and I can see in the output that it complains about not being able to find libfontconfig.so.1

Output received:
```
...
automatically fixing dependencies for ELF files
searching for dependencies of ./lib/monitor-x86_64/plugins/org.eclipse.equinox.launcher.gtk.linux.x86_64_1.1.200.v20120913-144807/eclipse_1502.so
searching for dependencies of ./lib/monitor-x86_64/monitor
searching for dependencies of ./lib/monitor-x86_64/libcairo-swt.so
  libXrender.so.1 -> found: /nix/store/2wgspjabaxpqlqx7xxlpii2s64fr1nxg-libXrender-0.9.10/lib/libXrender.so.1
  libXext.so.6 -> found: /nix/store/fnqkfjjxmiqlmdm3sf3xj23i8vrnsihx-libXext-1.3.3/lib/libXext.so.6
  libX11.so.6 -> found: /nix/store/4zcd8bkx7kyhs6ccnbrh9bgys8sbp6qy-libX11-1.6.7/lib/libX11.so.6
  libfontconfig.so.1 -> not found!
  libfreetype.so.6 -> found: /nix/store/fkf1kxgzhvca7zmcr76bv3sjvhbl0cdk-freetype-2.9/lib/libfreetype.so.6
builder for '/nix/store/60ikx0mrlvalspn9nlz7kbq2rnsmalpq-androidsdk.drv' failed with exit code 1
error: build of '/nix/store/60ikx0mrlvalspn9nlz7kbq2rnsmalpq-androidsdk.drv' failed
```

### Steps to reproduce

```
with (import <nixpkgs> {
  config = {
    allowUnfree = true;
    android_sdk.accept_license = true;
  };
});

let
  jdk = jdk8;
  androidComposition = androidenv.composeAndroidPackages {
    toolsVersion = "26.1.1";
  };
  androidSdk = androidComposition.androidsdk;
  androidPlatformTools = androidComposition.platform-tools;
in mkShell {
  buildInputs = [
    jdk
    androidSdk
    androidPlatformTools
  ];
}
```

## Technical details

- system: `"x86_64-linux"`
- host os: `Linux 4.20.3, NixOS, 19.03.git.6d4f3311d4b (Koi)`
- multi-user?: `yes`
- sandbox: `yes`
- version: `nix-env (Nix) 2.2`
- channels(root): `"nixos-18.09.1922.97e0d53d669"`
- channels(terje.larsen): `""`
- nixpkgs: `/home/terje.larsen/src/github.com/terlar/nix-config/nixpkgs`

I am running against a nixpkgs checkout at:
https://github.com/nixos/nixpkgs/tree/6d4f3311d4b5662e2703cfd3278fd78bd972a56c

**Comment by svanderburg** [comment]
This should have been fixed in revision 9130c1d592b74d52165b953f95ff8f6d1027632b

**Comment by terlar** [comment]
Thank you, that worked better.
